public/employers/mgm-resorts.yml

```diff
@@ -0,0 +1,42 @@
+name: MGM Resorts International
+aliases:
+  - MGM Grand, Inc.
+  - MGM Mirage
+shortName: MGM Resorts
+ticker: MGM
+wiki: MGM Resorts International
+image: mgm-resorts.svg
+officialWebsite: https://www.mgmresorts.com/en/company.html
+industries:
+  - hospitality
+  - lodging
+location:
+  city: Paradise
+  state: NV
+  country: us
+  international: true
+  wiki: Paradise, Nevada
+employeesBefore:
+  type: approximately
+  upperBound: 74500
+  year: 2020
+  yearQuarter: Q1
+summary: Marriott elected to suspend investment activity and implemented salary reductions for executives. However, this did not negatively affect customers, who were still able to perform changes and cancellations during the COVID-19 crisis.
```

Comment:
MGM do get investments from Marriott but I don't think the company is subsidiary of Marriott unless I'm wrong. I guess at first glance, this would imply a subsidiary relationship I think. This could be worded better to focus on MGM more.

Reply:
I think I copied the wrong line here. Will fix.